index.d.ts

```diff
@@ -2,7 +2,19 @@ declare module "typy" {
 
 	type SafeFunction = (...args: any[]) => any;
 
-	export default function typy (input: any, objectPath?: string): Typy
+	export default function t (obj: any, nestedKeys?: string): Typy;
```

Comment:
Functions `t` and `typy` are referring to the same function. We can rename/remove the declaration for `typy` function and just use `t` just like how you put here and also declare it as the default export. It will make it more consistent with the actual source code as well.

```js
export function t(input: any, objectPath?: string): Typy
export default t;
```